lib/uint64.js

```diff
@@ -0,0 +1,274 @@
+'use strict';
+
+const UINT32_MAX = 0xffffffff;
+
+const charToNum = {
+  0: 0, 1: 1, 2: 2,
+  3: 3, 4: 4, 5: 5,
+  6: 6, 7: 7, 8: 8,
+  9: 9,
+  a: 10, A: 10,
+  b: 11, B: 11,
+  c: 12, C: 12,
+  d: 13, D: 13,
+  e: 14, E: 14,
+  f: 15, F: 15,
+};
+
+class Uint64 {
+  constructor(value) {
+    this.value = new Uint32Array(2);
+    if (value instanceof Uint64) {
+      this.value[0] = value.value[0];
+      this.value[1] = value.value[1];
+      return;
+    }
+    const numValue = Number(value);
+    if (Number.isNaN(numValue) || numValue <= 0) {
+      return;
```

Comment:
`Number('hello') -> NaN` throw isn't a js way

Reply:
IMO this may lead to some hard to find bugs, i.e. if one of your computation operations result in a non-valid Uint32 (as per spec) value this will just make this Uint64 instance's value `0` and the computations will resume with this unexpected value instead of failing at this step with reasonable and easy to spot error.

Reply:
It will lead to the same kind of bugs as any operation on numbers that lead to `ToIntX` and `ToUintX` abstract operations usage since all of them have this behavior.
Same, if we use `value >>> 0` and `value` can't be converted to `int32` the result will be 0.

Reply:
Well, and I want to avoid having those bugs by either throwing or using some other mechanism that will help to search and identify them. If other people have such bugs it doesn't mean that we should. (As for `value` I just pointed that if we don't want to handle anything we may as well just use `value >>> 0` it doesn't help with this issue)